src/Chain.php

```diff
@@ -113,9 +113,9 @@ public function between($min, $max)
      *
      * @return $this
      */
-    public function bool()
+    public function boolean()
```

Comment:
@berry-langerak 

Renamed this in order to be consistent with the class name - shall we add 

```php
/**
 * @deprecated
*/
public function bool()
{
    return $this->boolean();
}
```

?